src/controllers/controller.js

```diff
@@ -0,0 +1,62 @@
+import User from '../models/model';
+
+class UsersController {
+  constructor() {
+    this.users = [];
+  }
+
+  createUser(login, password, age) {
+    const newUser = new User(login, password, age);
+    this.users.push(newUser);
+    return newUser;
+  }
+
+  showAllUsers() {
+    return this.users;
+  }
+
+  updateUser(id, login, password, age) {
+    const currentUser = this.getUser(id);
+    if (currentUser) {
+      currentUser.update(login, password, age);
+      return currentUser;
+    }
+
+    return false;
+  }
+
+  deleteUser(id) {
+    const currentUser = this.getUser(id);
+    if (currentUser) {
+      currentUser.delete();
+      return true;
+    }
+    return false;
+  }
+
+  getUser(id) {
+    const currentUser = this.users.find((user) => user.id === id);
+    if (!currentUser) {
+      return false;
+    }
+
+    return currentUser;
+  }
+
+  getAutoSuggestUsers(loginSubstring, limit) {
```

Comment:
Does not work. Also returns weird error
`A user with this ID was not found.`
Please check all the datapoints with postman before submitting the review

Reply:
I checked it multiple times and it works with the UUID, per the spec. For example, '/users/661216f9-6be5-411e-bd97-5e4578fec651' returns the User object.

<img width="579" alt="Screen Shot 2020-02-03 at 11 44 43 AM" src="https://user-images.githubusercontent.com/16273131/73686003-0ca12d80-467c-11ea-8084-9b4975b98b36.png">

Also, the current working branch for module 2 is Review Branch #6 and this functionality has been modified on Review Branch #7 for module 3 to work with DB primary keys.